prototype/index.html

```diff
@@ -1,248 +1,292 @@
-
-						<div class="posted text-right">
-							Posted on January 5, 2017
-						</div>
-					</p>
-
-				</div>
-
-			</div>
-			<!--post WITHOUT IMAGE ENDS !-->
-
-			<!--pagination !-->
-			<div class="text-center">
-				<div class="pagination">
-					<a href="#">&laquo;</a>
-					<a href="#" class="active">1</a>
-					<a href="#">2</a>
-					<a href="#">3</a>
-					<a href="#">4</a>
-					<a href="#">5</a>
-					<a href="#">6</a>
-					<a href="#">&raquo;</a>
-				</div>
-			</div>
-			<!-- pagination ends !-->
-		</div>
-		<!-- main content ends !-->
-
-		<!-- sidebar starts !-->
-		<div class="col-sm-3">
-
-			<div class="row">
-
-				<!-- for categories list sidebar section !-->
-				<div class="col-sm-12">
-					<div class="highlight-title">CATEGORIES</div>
-					
-					<div class="sidebar-content">
-						<a href="">Tech</a><hr>
-						<a href="">News</a><hr>
-						<a href="">Tutorials</a><hr>
-						<a href="">Sports</a>
-					</div>
-
-				</div>
-				<!-- for categories list sidebar section ends !-->
-
-				<!-- for rss subscription sidebar section !-->
-				<div class="col-sm-12">
-					<div class="highlight-title">RSS FEED</div>
-
-					<div class="sidebar-content">
-
-						Subscribe to our RSS feed today and be the first to
-						get new updates<br><br>
-
-						<div class="row">
-							<div class="col-sm-12">
-								<div class="input-group">
-									<input type="text" class="form-control">
-									<span class="input-group-btn">
-										<button class="btn btn-default" type="button">Subscribe</button>
-									</span>
-								</div><!-- /input-group -->
-							</div><!-- /.col-sm-6 -->		
-						</div>				
-
-					</div>
-				</div>
-				<!-- for rss subscription sidebar section ends !-->
-
-			</div>
-
-		</div>
-		<!-- sidebar ends !-->
-
-	</div>
-</div>
-
-<!--newsletter area !-->
-
-<div class="newsletter">
-	<h1 style="color: #ccc">MOST USED TAGS:</h1>
-
-	<br>
-
-	<a href="" class="btn btn-primary btn-lg">Poeople</a> <a href="" class="btn btn-success btn-lg">Sports</a>
-	<a href="" class="btn btn-warning btn-lg">Poeople</a> <a href="" class="btn btn-danger btn-lg">Sports</a>
-	<a href="" class="btn btn-primary btn-lg">Poeople</a> <a href="" class="btn btn-success btn-lg">Sports</a>
-
-	<br><br>
-
-	<a href="" class="btn btn-warning btn-lg">Poeople</a> <a href="" class="btn btn-danger btn-lg">Sports</a>
-	<a href="" class="btn btn-primary btn-lg">Poeople</a> <a href="" class="btn btn-success btn-lg">Sports</a>
-
-</div>
-
-<div class="footer">
-	&copy; Copyright 2017 - Nerdstein. All Rights Reserved.
-</div>
-
-<script type="text/javascript" src="files/js/jquery.js"></script>
-<script src="//netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js"></script>
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
```

Comment:
This file seems off to me. I'll ask on Slack :)

Reply:
We agreed to remove the "prototype" directory altogether. It's not helpful any more.